### PR TITLE
Remove scroll from body when modal is open

### PR DESCRIPTION
**Pull request by abierbaum**
Add .no-scroll to body on modal open
Remove .no-scroll from body on modal close

### CODE REVIEW

File: src/app/issue/issue_ctrl.js [hunk 1]
```diff
@@ -91,7 +91,12 @@ mod.controller('IssueCtrl', function($scope, $modal, $rootScope, trRepoModel, gh
             templateUrl  : "issue/issue_details.tpl.html"
          };
 
-         $modal.open(opts);
+         // todo: remove this setting when future version of $modal adds automatically
+         // - see: https://github.com/trestle-pm/trestle/issues/80
+         $rootScope.modalIsUp = true;
+         $modal.open(opts).finally(function() {
+            $rootScope.modalIsUp = true;
+         });
```
Comment (my8bird) [inline src/app/issue/issue_ctrl.js:99]: doesn't one of these need to set the value to false?

Comment (abierbaum) [inline src/app/issue/issue_ctrl.js:99]: Yes, the one in the infally should set to false.  Missed that one.

**Pull request merged**